specification/loadtestservice/LoadTestService.Management/routes.tsp

```diff
@@ -0,0 +1,141 @@
+import "./models.tsp";
+import "@azure-tools/typespec-providerhub";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.ResourceManager;
+using OpenAPI;
+using Autorest;
+using Azure.ResourceManager.Foundations;
+namespace Microsoft.LoadTestService.Draft;
+
+interface Operations extends Azure.ResourceManager.Operations {}
+
+/**
+ * A long-running resource CreateOrUpdate (PUT)
+ * @template TResource the resource being patched
+ * @template TBaseParameters Optional. Allows overriding the operation parameters
+ */
+@autoRoute
+@doc("Create a {name}", TResource)
+@extension("x-ms-long-running-operation", true)
+@extension(
+  "x-ms-long-running-operation-options",
+  {
+    `final-state-via`: "azure-async-operation",
+  }
+)
+@armResourceCreateOrUpdate(TResource)
+@put
+op ArmResourceCreateOrUpdateAsyncCustom<
+  TResource extends ArmResource,
+  TBaseParameters = BaseParameters<TResource>
+>(
+  ...ResourceInstanceParameters<TResource, TBaseParameters>,
+
+  @doc("Resource create parameters.")
+  @body
+  body: TResource,
+): ArmResourceUpdatedResponse<TResource> | ArmResourceCreatedResponse<TResource> | ErrorResponse;
+
+/**
+ * A synchronous resource action.
+ * @template TResource The resource being acted upon
+ * @template TRequest The request model for the action
+ * @template TResponse The response model for the action
+ * @template TBaseParameters Optional. Allows overriding the parameters for the operation
+ */
+@autoRoute
+@armResourceAction(TResource)
+@post
+@returnsDoc("ARM operation completed successfully.")
+op ArmResourceActionSyncCustom<
+  TResource extends ArmResource,
+  TRequest extends TypeSpec.Reflection.Model | void,
+  TResponse extends TypeSpec.Reflection.Model | void,
+  TBaseParameters = BaseParameters<TResource>
+>(
+  ...ResourceInstanceParameters<TResource, TBaseParameters>,
+
+  @doc("The content of the action request")
+  @body
+  QuotaBucketRequest: TRequest,
+): TResponse | ErrorResponse;
+
+@armResourceOperations
+interface LoadTests {
+  listBySubscription is ArmListBySubscription<LoadTestResource>;
+  listByResourceGroup is ArmResourceListByParent<LoadTestResource>;
+  get is ArmResourceRead<LoadTestResource>; 
+  createOrUpdate is ArmResourceCreateOrUpdateAsyncCustom<LoadTestResource>;
+  update is ArmResourcePatchAsync<LoadTestResource, LoadTestProperties>;
+  delete is ArmResourceDeleteAsync<LoadTestResource>;
+  
+  #suppress "@azure-tools/typespec-azure-core/no-operation-id" "OperationId in already published GA spec is different from the auto generated one"
+  @operationId("LoadTests_ListOutboundNetworkDependenciesEndpoints")
+  outboundNetworkDependenciesEndpoint is ArmResourceListByParent<OutboundEnvironmentEndpoint>;
```

Comment:
What is this API? Where do we use it?

Reply:
This API returns a json with list of endpoints our batch VMs connect to in private load test. This is a common API in other services as well.